apps/src/templates/studioHomepages/BorderedCallToAction.jsx

```diff
@@ -25,82 +25,36 @@ const BorderedCallToAction = ({
   const borderStyle = solidBorder ? styles.solidBorder : styles.dashedBorder;
 
   return (
-    <div style={{...styles.outerBox, ...borderStyle}} className={className}>
-      <div style={styles.wordBox}>
-        <div style={styles.heading}>{headingText}</div>
-        <div style={styles.description}>{descriptionText}</div>
+    <div className={`${styles.outerBox} ${borderStyle} ${className}`}>
+      <div className={styles.textWrapper}>
+        <Heading3 visualAppearance="heading-sm">{headingText}</Heading3>
+        <BodyThreeText>{descriptionText}</BodyThreeText>
       </div>
       <Button
-        onClick={onClick || (() => navigateToHref(buttonUrl))}
+        onClick={onClick}
         className={buttonClass}
         color={buttonColor}
+        size={'s'}
+        type={buttonType}
         text={buttonText}
-        style={styles.button}
+        href={buttonUrl}
+        useAsLink={!!buttonUrl}
```

Comment:
Oooo interesting... this is the first time I'm realizing the button was designed this way. (This is not on you but I'm surprised we would have a state where we provide an href value but *don't useAsLink)

Reply:
Oh that's a great point! I just tried to see if it would automatically set `useAsLink` if there's a `buttonUrl` and it does not and gives an error.